samples/hello-android/build.gradle.kts

```diff
@@ -1,15 +1,15 @@
 plugins {
-    id("com.android.application") version "2.3.3"
+    id("com.android.application") version "3.0.0"
     kotlin("android") version "1.1.51"
 }
 
 android {
-    buildToolsVersion("25.0.0")
-    compileSdkVersion(23)
+    buildToolsVersion("27.0.0")
+    compileSdkVersion(26)
```

Comment:
You can target 27 for `compileSdkVersion` and `targetSdkVersion`.

Reply:
Right. I purposely targeted API 26, aka Android 8, aka Oreo, as this is the latest released version. I suppose we could target API 27 already.